### ISSUE TITLE
'require' and 'define' are being transformed in the public folder

### ISSUE THREAD

**Issue by ggayowsky** [Esri/ember-cli-amd#40]
In version, 0.4.9 of the application. The regex that transformed `require` and `define` were only applied to the app, tests, and vendor files.

In the newest addon, the public folder is also transformed which breaks applications that self host the esri libraries.

**Comment by ffaubry** [comment]
Could be more specific? A small repo to reproduce?
Your pattern is unknown from us.
We don't understand why esri libraries would be under the public folder.

**Comment by ggayowsky** [comment]
This is related to #23 

A small repo would be hard to make however I will try to explain our app / use case.

Our application needs to be deployed in places without an outside internet connection, so I cannot rely on getting the arcgis files from the CDN (e.g. `https://js.arcgis.com/3.27/`).

So our application has downloaded the files (which can be done from the esri developer site) and bundles it with the application under the /public/arcgis folder.

The css is included in the HTML files as so:

```
<link rel="stylesheet" href="{{rootURL}}arcgis/esri/css/esri.css">
```

and the amd configuration in the ember-cli-build.js

```
amd: {
  loader: '/arcgis/init.js',
  packages: [
    'esri', 'dojo', 'dojox', 'dijit', 'put-selector', 'xstyle', 'dgrid'
  ],
  configPath: 'config/dojo-config.js',
  locale: 'en-us'
}
```

with a minor tweak to the init.js file to point to the locally hosted dojo, up until version 0.4.9, we could use this addon successfully with the bundled ArcGIS code. At 1.0.0 this functionality broke, because the transform done on `require` and `define` is now applied to all files in the public folder as well (which I believe is a bug on its own... especially if a public file were trying to use dojo require).

**Comment by ffaubry** [comment]
@odoe I don't think the jsapi should be deployed under public in this case?!

**Comment by ffaubry** [comment]
This is fixed in 2.0.
Build sub-directories can be excluded from the conversion
```
amd: {
  excludePaths: ['assets/arcgis']
}
```
